report/2.Analysis/ArduinoInEducation.tex

```diff
@@ -0,0 +1,88 @@
+There are a multiple of courses, one of these regard electronics, where many schools choose to utilize the Arduino platform\footnote{based on the fact that 4 of the students in this group studied at HTX in different parts of Denmark and all utilized the Arduino platform in this course. 
+The HTX institution contacted by the group also utilize the Arduino platform}\cite{holstebro_education}.0
+
+This technically oriented course, often called electronics\footnote{The name varies between institutions}, aims to familiarize the students with electrical components and circuits as well as basic programming.\cite{holstebro_electronic}
+Arduino is an ideal platform for this, as it is possible to connect I/O devices to the Arduino micro-controller, which in turn can be programmed to do simple tasks. 
+This course is then used to spike the interest of the student regarding engineering fields, such as programming or electronics.
+
+
+\subsection{Aalborg Tekniske Gymnasium Electronic Course}
+In relation to the discipline of understanding the initial problem, the project group contacted one of the major HTX institutions in North Jutland, Denmark, Namely Aalborg Tekniske Gymnasium\footnote{Aalborg Technical Gymnasium, https://aatg.dk/}. 
+The following subsection will summarize the important and noticeable aspects of the electronic course conducted on Aalborg Tekniske Gymnasium. 
+The conclusions reached in this subsection are purely based on the statements made by the students and the teacher of this specific electronics course.
+The notes, from where the below conclusions were derived, can be found in the Appendix 1 on page \pageref{Interviews}.
+
+The electronics course itself does not teach the students programming directly. 
+Instead it focuses on project based arduino development. 
+Educational programming experience of the students (if any), originates from another optional course called "Programming". 
+The Programming course teaches the students the basic constructions and terminology of general programming. 
+The students are taught C{\#} in the programming course. 
+
+The electronics course focuses on engaging the students in making arudino projects, not teaching the students the theory behind the projects. 
+This means that the students both have a very limited understanding of the arduino platform's architecture and a low understanding of the arduino language as explained earlier in the report, which they utilize in the electronics course on Aalborg Tekniske Gymnasium.
+
+Because of all this the electronics course has a relatively high learning curve. 
+Which results in a high number of students abandoning the course in favor of some of the courses with a more defined structures.
```

Comment:
Enten skal "Which" ændres til "this", eller punktummet før skal laves om til komma.